### PR TITLE
14 crear pantalla de opciones con tabs para mostrar distintas pantallas

### LINKED ISSUE #14: Crear pantalla de opciones con tabs para mostrar distintas pantallas.

### PR DESCRIPTION
**Pull request by frantorres**

### CODE REVIEW

**Review by AlejandroAkbal — CHANGES_REQUESTED** [review]
Needs to finish functionality

File: admin/class-animalshelter-menupage-animalshelter.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+			$this->nonce();
+			submit_button();
+		}
+	}
+
+	public function save(): void {
+		if ( 'animals' === $this->tab ) {
+			// Save input
+		} elseif ( 'breeds' === $this->tab ) {
+			// Save input
```
Comment (AlejandroAkbal) [inline admin/class-animalshelter-menupage-animalshelter.php:76]: Here

**Review by AlejandroAkbal — CHANGES_REQUESTED** [review]

File: admin/class-animalshelter-menupage-animalshelter.php (hunk 1 above)
Comment (frantorres) [reply admin/class-animalshelter-menupage-animalshelter.php:76]: For now it is a placeholder pointing the way for future changes.

File: admin/class-animalshelter-menupage.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	}
+
+	public function init(): void {
+		add_action( 'admin_menu', array( $this, 'register_sub_menu' ) );
+		//add_action( 'admin_enqueue_scripts', array( $this, 'register_css' ) );
+		//add_action( 'admin_enqueue_scripts', array( $this, 'register_js' ) );
+	}
+
+	public function register_js( $hook ): void {
+		wp_enqueue_media();
```
Comment (AlejandroAkbal) [inline admin/class-animalshelter-menupage.php]: What is this used for?

File: admin/class-animalshelter-menupage.php [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+		?>
+		<h3><?php echo esc_html( $title ); ?></h3>
+		<?php
+	}
+
+	public function show_description( $description ): void {
+		?>
+		<p><?php echo esc_html( $description ); ?></p>
+		<?php
+	}
```
Comment (AlejandroAkbal) [inline admin/class-animalshelter-menupage.php:53]: Try to use the same style everywhere

Comment (frantorres) [reply admin/class-animalshelter-menupage.php:53]: Done!

File: admin/class-animalshelter-menupage.php (hunk 2 above)
Comment (frantorres) [reply admin/class-animalshelter-menupage.php]: Removed thanks!

**Comment by AlejandroAkbal** [comment]
There is a conflict, everything else looks good!

**Review by AlejandroAkbal — APPROVED** [review]

**Pull request merged**